packages/swarm-components/src/TogglePill.jsx

```diff
@@ -1,24 +1,38 @@
+// @flow
 import * as React from 'react';
 
 export type Props = {
+  /**
+   * Indicates whether the toggle is selected
+   */
   checked: boolean,
```

Comment:
Should this be required? I'd assumed if there's no checked prop you can default to false.